corehq/toggles.py

```diff
@@ -849,6 +849,13 @@ def _commtrackify(domain_name, toggle_is_enabled):
     [NAMESPACE_DOMAIN],
 )
 
+FORMPLAYER_USE_LIVEQUERY = StaticToggle(
+    'use_live_query',
```

Comment:
not sure if this is blocking or not, but I think it's convention to have the slug be the lowercase version of the variable name, so this should be `'formplayer_use_livequery'`